mlflow/tracking/fluent.py

```diff
@@ -521,10 +835,50 @@ def list_run_infos(
     :param experiment_id: The experiment id which to search
     :param run_view_type: ACTIVE_ONLY, DELETED_ONLY, or ALL runs
     :param max_results: Maximum number of results desired.
-    :param order_by: List of order_by clauses.
+    :param order_by: List of order_by clauses. Currently supported values are
+           are ``metric.key``, ``parameter.key``, ``tag.key``, ``attribute.key``.
+           For example, ``order_by=["tag.release ASC", "metric.click_rate DESC"]``.
 
     :return: A list of :py:class:`mlflow.entities.RunInfo` objects that satisfy the
         search expressions.
+
+    .. code-block:: python
+        :caption: Example
+
+        from pprint import pprint
+        import mlflow
+
+        # Get run info for experiment id 0, include only active runs
+        pprint("RunInfo: {}".format(mlflow.list_run_infos(str(0), run_view_type=1)))
+
+        print("-" * 80)
+
+        # Get run info for experiment id 0, include all runs
+        # order_by types are ['metric.key', 'parameter.key', 'tag.key', 'attribute.key']
```

Comment:
I think this comment (`# order_by types are ['metric.key', 'parameter.key', 'tag.key', 'attribute.key']`) is not necessary because the `order_by` parameter description explains the same thing.